communities/microgalaxy/lab/sections/2_microbial_isolates.yml

```diff
@@ -301,15 +301,81 @@ tabs:
       - id: genomics
         title: Genomics Workflows
         content:
+          - title_md: Genome annotation workflow with Helixer
```

Comment:
Can you put first the Control quality, Genome Assembly and Genome Annotation workflows?